scrapi/harvesters/lwbin.py

```diff
@@ -0,0 +1,155 @@
+from scrapi.base.helpers import build_properties
+
+
+logger = logging.getLogger(__name__)
+
+
+def process_contributors(authors, email):
+    """Process authors and add author emails
+    """
+
+    authors = authors.strip().replace('<span class="author-names">', '').replace('</span>', '')
+    authors = authors.split(',')
+
+    if ' and ' in authors[-1] or ' <em>et al.</em>' in authors[-1]:
+        split_name = authors.pop(-1).replace(' <em>et al.</em>', '').split(' and ')
+        authors.extend(split_name)
+
+    contributor_list = []
+    for person in authors:
+        name = HumanName(person)
+        contributor = {
+            'name': person,
+            'givenName': name.first,
+            'additionalName': name.middle,
+            'familyName': name.last,
+            'email': email,
+            'sameAs': [],
+        }
+        contributor_list.append(contributor)
+
+    return contributor_list
+
+
+def process_licenses(license_title, license_url):
+    """Process licenses to comply with the noormalized schema
+    """
+
+    license = {
+        'uri': license_url or "",
+        'description': license_title or ""
+    }
+
+    return [license]
+
+
+def construct_url(url, dataset_path, end_point):
+    """Return a url that directs back to the page on LBWIN Data Hub instead of the source page.
+
+    Keyword arguments:
+    url -- host url
+    dataset_path -- parent path of all datasets
+    end_point -- name of datasets
+    """
+
+    return "/".join([url, dataset_path, end_point])
+
+
+class LWBINHarvester(JSONHarvester):
+    short_name = 'lwbin'
+    long_name = 'Lake Winnipeg Basin Information Network'
+    url = 'http://130.179.67.140'
+    dataset_path = "dataset"   # dataset base url for constructing urls that go back to LWBIN instead of source pages.
+
+    DEFAULT_ENCODING = 'UTF-8'
+
+    record_encoding = None
+
+    @property
+    def schema(self):
+        return {
+            'title': ('/title', lambda x: x if x else ''),
+            'description': ('/notes', lambda x: x[0] if (isinstance(x, list) and x) else x or ''),
+            'providerUpdatedDateTime': ('/metadata_modified', lambda x: parse(x).isoformat()),
+            'uris': {
+                'canonicalUri': ('/name', lambda x: construct_url(self.url, self.dataset_path, x)),  # Construct new urls directing to LWBIN
+                'objectUris': '/url'  # Default urls from the metadata directing to source pages
+            },
+            'contributors': ('/author', '/author_email', process_contributors),
+            'licenses': ('/license_title', '/license_url', process_licenses),
+            'tags': ('/tags', lambda x: [tag['name'].lower() for tag in (x or [])]),
+            'freeToRead': {
+                'startDate': ('/isopen', '/metadata_created', lambda x, y: parse(y).date().isoformat() if x else None)
+            },
+            'otherProperties': build_properties(
+                ('licenseTitle', '/license_title'),
```

Comment:
This is already present in the licenses metadata, so you can remove it from here